admin/c2cgeoportal_admin/views/layers_wms.py

```diff
@@ -0,0 +1,56 @@
+from pyramid.view import view_defaults
+from pyramid.view import view_config
+from c2cgeoform.views.abstract_views import AbstractViews
+from c2cgeoportal_commons.models.main import LayerWMS
+from colanderalchemy import setup_schema
+from c2cgeoform.views.abstract_views import ListField
+
+setup_schema(None, LayerWMS)
+
+
+@view_defaults(match_param='table=layers_wms')
+class LayerWmsViews(AbstractViews):
+    _list_fields = [
+        ListField('name'),
+        ListField('metadata_url'),
+        ListField('description'),
+        ListField('public'),
+        ListField('geo_table'),
+        ListField('exclude_properties'),
+        ListField('layer'),
+        ListField('style'),
+        ListField('time_mode'),
+        ListField('time_widget'),
+        ListField('ogc_server', renderer=lambda wmsLayer: wmsLayer.ogc_server.name),
```

Comment:
convention in python is to use _ instead of upper case letters in variable names : wms_layer

Reply:
note that layer_wms may be more consistent with the model.